qemu/tests/x86_cpu_flag_disable.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from virttest import error_context, env_process
+from qemu.tests.x86_cpu_model import check_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flags.
+    1) Check if current flags are in the supported lists, if no, cancel test
+    2) Otherwise, boot guest with the cpu flags disabled
+    3) Check cpu flags inside guest(only for linux guest)
+    4) Check kvmclock inside guest(only for linux guest)
+    4) Reboot guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    flags = params["flags"]
+    check_host_flags = params.get_boolean("check_host_flags")
+    if check_host_flags:
+        check_flags(params, flags, test)
+
+    params["start_vm"] = "yes"
+    vm_name = params['main_vm']
+    env_process.preprocess_vm(test, params, env, vm_name)
+
+    vm = env.get_vm(vm_name)
+    cmd = params["check_flag_cmd"]
+    error_context.context("Try to log into guest", logging.info)
+    session = vm.wait_for_login()
+    vm_flags_out = session.cmd_output(cmd).split()
+    if flags[0] in vm_flags_out:
+        test.fail("Flag %s shouldn't in guest" % flags[0])
+
+    check_clock = params.get("check_clock")
+    vm_clock_out = session.cmd_status_output(check_clock)[1]
+    if 'kvmclock' in vm_clock_out:
+        test.fail("kvmclock shouldn't be found")
+
+    if params.get("reboot_method"):
```

Comment:
Seems this judgement could be removed, this is no other condition.

Reply:
> Seems this judgement could be removed, this is no other condition.

Thanks and update.